src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1197,6 +1198,11 @@ static ucs_status_t uct_ib_mlx5dv_check_dc(uct_ib_device_t *dev)
         goto out_destroy_qp;
     }
 
+    /* ibv_set_ece check whether ECE is supported */
+    if ((ibv_query_ece(qp, &ece) == 0) && (ibv_set_ece(qp, &ece) == 0)) {
+        dev->flags |= UCT_IB_DEVICE_FLAG_ECE;
+    }
```

Comment:
don't we already have uct_ib_device_ece_check() function?

Reply:
@yosefe Yes. There's already ```uct_ib_device_ece_check``` help function could check it.
Here, since there's already created dummy QP here, it reuses this QP to check whether ECE operation is supported.
Should we always use ```uct_ib_device_ece_check``` to check whether ECE operation is supported?

Reply:
> Should we always use `uct_ib_device_ece_check` to check whether ECE operation is supported?

yes

Reply:
Done